src/v1/constants.ts

```diff
@@ -0,0 +1,12 @@
+export const constants = {
+  /***********************************
+   *	Static Urls & Variables
+   ************************************/
+  PRODUCT_NAME: "Core Framework",
+};
+
+export const max_ticket_agent = 500;
+export const max_ticket_user = 100;
+export const rate_train_one = 4;
```

Comment:
If this specifies the rates for the express, and local train, then kindly give the variable names according to it.